app/src/main/java/ca/marshallasch/veil/PostListAdapter.java

```diff
@@ -33,15 +29,15 @@
     /**
      * This is the cell for each post in the list.
      */
-    public static class ViewHolder extends RecyclerView.ViewHolder {
+     static class ViewHolder extends RecyclerView.ViewHolder {
         private TextView title;
         private TextView contentPreview;
 
         /**
          * constructor for the ViewHolder class
          * @param itemsView the XML layout for the cell. Currently is a list_cell.xml
          */
-        public ViewHolder(View itemsView){
+        ViewHolder(View itemsView){
```

Comment:
Default is private and makes sense since it only used in this class. But we still should add the private keyword just to improve readability.

Reply:
default is package-private not private, it does not work if it is private. since it is used in the parent class